<launch>
    <node pkg="sinwave" name="generator" type="generator.py" output="screen">
        <param name='outTopic' type='string' value='sine1'/>
        <param name='frequency' type='double' value='1'/>
        <param name='executionFrequency' type='double' value='30.0'/>
    </node>

    <node pkg="sinwave" name="absolute" type="absolute.py" output="screen">
      <param name='in_topic' type='string' value='sine1'/>
      <param name='out_topic' type='string' value='sine2'/>
    </node>

    <node name="loggerRqt" pkg="rqt_plot" type="rqt_plot" args="/sine1/value /sine2/value"/>
</launch>
